website/docs/faq.md

```diff
@@ -4,4 +4,78 @@ title: FAQ
+We can connect with SSL if certificate is optional. If our requirement is ca.cert auth for the SSL connection, then this is not available on portal. 
+
+### Is there any way to use Litmus within github? Basically when someone submits a k8s deployment for a PR, We want to run chaos Experiment on that to see whether it passes or not.
+
+We can use it with the help of github-chaos-action. Using github-chaos-action we can automate the chaos execution on an application in the same place where the code is stored. We can write individual tasks along with chaos actions and combine them to create a custom GitHub workflow. GitHub Workflows are custom automated processes that we can set up in our repository to build, test, package, or deploy any code project on GitHub. Including the Github chaos actions in our workflow YAML, We can test the performance/resiliency of our application in a much simpler and better way. To know more please visit  our Github chaos action [repository](https://github.com/litmuschaos/github-chaos-actions).
+
+### I encountered the concept of `namespace` and `cluster` scope during the installation. What is meant by the scopes, and how does it affect experiments to be performed outside or inside the litmus Namespace?
+
+The scope of portal control plane (portal) installation tuned by the env `PORTAL_SCOPE` of litmusportal-server deployment can be kept as a namespace if you want to provide a very restricted access to litmus; It's useful in dev environments like Okteto cloud etc.
+That basically restricts portal installation along with its agent to a single namespace and the chaos operator, exporter all get installed in a single namespace and can only perform and monitor chaos in that namespace.
+Other than that there is another key in the control plane’s configmap `litmus-portal-admin-config` called  `AgentScope`, this is given to allow users to restrict access to the litmus self-agent components self-agent is the agent for your control plane cluster (exporter, operator etc), you can use both of them in a way to give access as per the requirement.
+The above holds for the control plane and self agent mainly, for the external agents which can be connected using the CLI, litmusctl you can provide the scope of the agent while using the utility to connect your other cluster to the control plane with access to just a single namespace or cluster-wide access.
+using a combination of AgentScope: cluster and `PORTAL_SCOPE`  env set to cluster would give you cluster admin privileges to inject chaos on all namespaces where control plane/portal is installed. For external agents just selecting the scope of installation as cluster would be sufficient via litmusctl.
+
+### What does failed status of workflow means in LitmusPortal?
+
+Failed status indicates that the default hypothesis of experiment was disproved and some of the experiments in the workflow failed, In such case the resiliency score will be less than 100%.
```

Comment:
Failure could also be due a a misconfigured step or error in argo workflows besides the normal experiment failures

Reply:
Is It ok?
Failed status indicates that either there is some misconfiguration in the workflow or the default hypothesis of the experiment was disproved and some of the experiments in the workflow failed, In such a case, the resiliency score will be less than 100.